packages/esm-patient-conditions-app/src/conditions/conditions-detailed-summary.component.tsx

```diff
@@ -24,39 +25,57 @@ function ConditionsDetailedSummary({ patient }) {
   const { t } = useTranslation();
   const displayText = t('conditions', 'Conditions');
   const headerTitle = t('conditions', 'Conditions');
+  const [filter, setFilter] = useState('');
```

Comment:
Three things here:

1. I think we should default the condition list to "Active" conditions instead of "All". In this context, "Active" implies "is part of the patient's care", so it's much more important that those be shown.
2. We know what the default is, so we may as well use that.
3. I think stricter typing is better here to prevent future errors.

```suggestion
  const [filter, setFilter] = useState<'All' | 'Active' | 'Inactive'>('Active');
```